routes/files.js

```diff
@@ -0,0 +1,38 @@
+const Joi = require('joi');
+
+const definitions = require('./__swagger__/definitions');
+const prefix = require('./__utils__/path')('/files');
+
+const routes = [];
+
+const filesMock = [
+  {
+    file: 'Datei:Sunday 3 Besetzung (retouched).jpg',
+    url: 'https://de.wikipedia.org/wiki/Datei:Sunday_3_Besetzung_(retouched).jpg',
+  },
+];
+
+routes.push({
+  path: prefix(''),
```

Comment:
Just a nitpick: I find it a bit weird to have the actual path of this route in the header statement instead of simply passing it as a plain string here. We still can add a prefix later on if we need it!

Reply:
✅ done